tests/unit/training/test_sae_trainer.py

```diff
@@ -204,7 +204,6 @@ def test_train_sae_group_on_language_model__runs(
     checkpoint_dir = tmp_path / "checkpoint"
     cfg = build_sae_cfg(
         checkpoint_path=str(checkpoint_dir),
-        train_batch_size=32,
```

Comment:
Why was this changed?

Reply:
For similar reasons as in [this comment](https://github.com/jbloomAus/SAELens/pull/301/files#r1776064470). `train_batch_size` isn't an attribute of LanguageModelSAERunnerConfig.